examples/rust/examples/maker/maker.rs

```diff
@@ -1,21 +1,19 @@
 use crate::settings::Settings;
-use crate::tracing_utils::{get_subscriber, init_subscriber};
-use backon::{ExponentialBuilder, Retryable};
 use ethers::abi::{AbiEncode, RawLog};
 use ethers::prelude::{
     rand::{thread_rng, Rng},
-    Address, EthLogDecode, Http, JsonRpcClient, LocalWallet, Middleware, Provider, Signer,
+    Address, EthLogDecode, Http, Ipc, JsonRpcClient, LocalWallet, Middleware, Provider, Signer,
     SignerMiddleware, Ws, U256,
 };
 use ethers::utils::keccak256;
 use http::Uri;
+use log::{error, info, warn};
```

Comment:
Let's use tracing.

Reply:
So I avoided tracing since the output is not as easy to review when running as an example (so I wouldn't expect anything to be required to ingest and decode logs. So this logging change moved the trace from:
```
{"v":0,"name":"trade-api-maker","msg":"Re/starting maker","level":40,"hostname":"Moria","pid":19125,"time":"2023-10-22T23:10:45.200577289Z","target":"maker","line":57,"file":"examples/maker/maker.rs"}
{"v":0,"name":"trade-api-maker","msg":"Maker has authenticated with Valorem","level":30,"hostname":"Moria","pid":19125,"time":"2023-10-22T23:10:48.394038536Z","target":"maker","line":735,"file":"examples/maker/maker.rs"}
{"v":0,"name":"trade-api-maker","msg":"Ready for RFQs from Takers","level":30,"hostname":"Moria","pid":19125,"time":"2023-10-22T23:10:49.222303679Z","target":"maker","line":162,"file":"examples/maker/maker.rs"}
```

to 

```
 2023-10-22T23:12:38.741Z INFO  maker > Maker has authenticated with Valorem
 2023-10-22T23:12:39.978Z INFO  maker > Ready for RFQs from Takers
```

This is far more better to read while running the example.

Not sure what you mean by consistency here, the Taker example uses println and the library doesn't have any logging.

Reply:
in that case leave as is, i meant by trade api, but you are correct for the examples